core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportExtendedMutationData.java

```diff
@@ -292,8 +293,16 @@ private void checkGermlineMutations() throws DaoException {
 
     private void checkMutationCounts() throws DaoException {
 
-        ClinicalData clinicalData = DaoClinicalData.getDatum("study_tcga_pub", "TCGA-AA-3664", "MUTATION_COUNT");
```

Comment:
This seemed to have been checking `MUTATION_COUNT` using a patient id before. Not sure how this was working before, gets a null exception now. I think `MUTATION_COUNT` should only be at the sample level, so seems ok to change